### ISSUE TITLE
Can we have properties converted to Title Case to match Freecad style?

### ISSUE THREAD

**Issue by leoheck** [shaise/FreeCAD_SheetMetal#344]
Hey @shaise nice to meet you here. I did not know that this was also your work. 
What do you think about updating property names on this Workbench too?

![image](https://github.com/shaise/FreeCAD_SheetMetal/assets/1277920/0cc75e49-875e-438e-b508-d0839da4cac4)

**Comment by shaise** [comment]
Yes, I had a feeling it will come into this. Sheetmetal is a bit more complicated in that matter. But I will take a look as well.

**Comment by leoheck** [comment]
> Yes, I had a feeling it will come into this.

Sorry to give you this much work dude!

**Comment by luzpaz** [comment]
I guess there would need to be a slow deprecation process whereby the deprecated properties need to be converted by a script for backward compatibility, notify the user that older versions of SheetMetal addon aren't compatible... ?

**Comment by leoheck** [comment]
What if the priperties are read from the file as case insensitive, and converted to the title case internally. When saving the file, the title case is used?

**Comment by shaise** [comment]
I think that I will probably end up doing the same as I did in Fasteners. Only reason I postponed it is because I need to see how I merge the changes that were done for Ondsel's sheetmetel.

**Comment by luzpaz** [comment]
@shaise is this still relevant ?

**Comment by shaise** [comment]
Actually, with my new changes, it is rather easy now to change parameter names and still be compatible with old files (it will rename them as soon as a change is made)
However, since there is a much more comfortable UI Task Panel, I'm not sure it is necessary.

**Comment by leoheck** [comment]
> UI Task Panel

Which UI Task Panel?

**Comment by shaise** [comment]
with the new version, there is a task panel for this function

**Comment by leoheck** [comment]
The new version of Freecad or the Workbench? So no properties on the properties panel anymore?

**Comment by shaise** [comment]
new version of sheetmetal:
![image](https://github.com/user-attachments/assets/d565017a-1e44-4f40-8376-e3d18d436be7)

**Comment by leoheck** [comment]
Nice, and those settings won't be visible on the properties panel?

**Comment by shaise** [comment]
the property panel is the low level of any feature and will always work. The UI is only a helper.

**Comment by leoheck** [comment]
But can we have the title case on the property panel as well to mimic other properties? This will give the professional look that Freecad deserves.

**Comment by shaise** [comment]
@luzpaz , what do you think?
